src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -27,8 +27,22 @@ public AuthOps(DatastoreService ds) {
     UserController controller = new UserController(ds);
   }
 
-  // generates cookie. Used in most methods.
-  public Cookie generateCurrCookie(HttpServletRequest request, HttpServletResponse response) {
+  public String getAuthID() {
+    // call auth service to generate AuthID
+    UserService userService = UserServiceFactory.getUserService();
+    String AuthID = userService.getCurrentUser().getUserId();
```

Comment:
There wouldn't always be a current user (eg. if user is not logged in), so this might error out with a nullpointerexception.  getAuthID then maybe can return a "not-logged-in" state for when the user is not logged in